pkg/cmd/root/root_test.go

```diff
@@ -0,0 +1,104 @@
+package root
```

Comment:
This is not great for visibility. 

We can call it:

- charmil/rules
- cobralint

etc.